examples/gocean/eg5/profile/README.md

```diff
@@ -0,0 +1,130 @@
+# PSyclone GOcean PSyData Example 2 - Profiling
+
+**Author:** J. Henrichs, Bureau of Meteorology
+
+## Introduction
+
+This is a very simple test that shows how to use the profiling
+support in PSyclone. It is a stand alone program that can be compiled
+and run. 
+
+## Compilation
+The makefile supports compiling and linking with the following PSyclone
+profile wrapper libraries:
+- template
+- simple_timing
+- dl_timer
+- drhook
+- lfric
+
+By default (``make`` without an argument) the ``template`` library will 
+be used, which just prints the name of the regions called.
+In order to test any of the other libraries, just use the
+command ``make <wrapper library name>`` and use the name listed above
+for ``<wrapper library name>``. The name of the executable will be
+``profile_test.<wrapper library name>``. There is also a target ``make all``
+which will create executables for all libraries listed above.    
+
+You have to compile the GOcean infrastructure library
+dl_esm_inf, and the corresponding profile wrapper library in
+``lib/profiling``. By default, the compilation uses the version
+of the dl_esm_inf library provided as a git submodule (under
+``external/dl_esm_inf ``- see
+https://psyclone-dev.readthedocs.io/en/latest/working_practises.html)
+within the PSyclone repository (set the environment variable``INF_DIR``
+for the ``make`` command to pick a different version). The default build
+uses the "template" profiling library in ``lib/profiling/template``.
+More detailed instructions for compiling these libraries are are given in
+the corresponding subdirectories.
+
+If you are using ``dl_timer`` or ``drhook``, you need to compile these
+libraries yourself first, and modify the ``Makefile`` in this directory
+to specify the required linking parameters. The ``Makefile``
+supports the following environment variables that can be defined
+to find the various software packages:
+
+### INF_DIR:
+The location of the dl_esm_inf infrastructure library, it defaults to
+``../../../../external/dl_esm_inf/finite_difference``,
+which is the version included in PSyclone.
+### DL_TIMER_ROOT:
+The location of the apeg-dl_timer library. It defaults to
```

Comment:
s/apeg-dl_timer/dl_timer/ and please add a link to https://bitbucket.org/apeg/dl_timer/src/master/

Reply:
Done.